src/components/shared/NavigationBurger.vue

```diff
@@ -1,15 +1,14 @@
 <template>
-	<a
+	<button
 		class="navigation-burger"
 		:class="{ 'active': active }"
-		role="button"
 		aria-label="menu"
```

Comment:
We should make this label translatable, it sounds funny when my German VoiceOver says "menu"